src/components/vive-controls.js

```diff
@@ -94,6 +94,7 @@ module.exports.Component = registerComponent('vive-controls', {
     el.addEventListener('touchstart', this.onButtonTouchStart);
     el.addEventListener('model-loaded', this.onModelLoaded);
     el.addEventListener('axismove', this.onAxisMoved);
+    this.controlllerEventsActive = true;
```

Comment:
I see that `this.removeEventListeners` is called on `pause` but `this.addEventListeners` is not called in play. We should probably call it there.